cypress/e2e/tables-unified-search.cy.js

```diff
@@ -75,7 +77,7 @@ describe('The Home Page', () => {
 	})
 
 	it('Search for shared table via group share', () => {
-		cy.login({ userId: 'user1', password: '1234561' })
+		cy.login(alice)
```

Comment:
It would be even better to just create those users with different non-conflicting user ids during the before hook

That way there is no dependency on existing setup or configuration on the server used for testing

Examples using the cypress helper library:
https://github.com/nextcloud/richdocuments/blob/a76639cb09f39c7566c1ced6c3e330eb261e77bc/cypress/e2e/share-link.js#L28-L30
https://github.com/nextcloud/richdocuments/blob/a76639cb09f39c7566c1ced6c3e330eb261e77bc/cypress/support/commands.js#L24C14-L24C14